### ISSUE TITLE
aliases command don't ovverride

### ISSUE THREAD

**Issue by SuperAlecs** [LegendOfMCPE/EssentialsPE#292]
the aliases command such "whisper, gm" don't ovverride. sorry for my bad english!

**Comment by Sandertv** [comment]
Why don't they override? Is there any error?

**Comment by SuperAlecs** [comment]
the aliases command don't ovverride it give pocketmine's command not essentials, but the default command of essentials work such "tell"

**Comment by Sandertv** [comment]
Okay so what exactly is happening? (and isn't) Could you give an example of what you experience?

**Comment by SuperAlecs** [comment]
I run aliases (/w) and it run pocketmine command. Not essentialsPE and with pocketmine command i can't use (/r) for quickreply

**Comment by Sandertv** [comment]
What do you use to set aliases? (/r already is an alias automatically by the way)

**Comment by Sandertv** [comment]
Okay just to clarify: 
The gamemode stuff, there's no alias '/gm' only '/gmc, /gms, /gma' etc.
/r alias and the /reply command is made in EssentialsPE
/w alias? I don't know, doesn't exist that I know of.

**Comment by SuperAlecs** [comment]
i say the alias command of essentialspe must ovverride the pocketmine command but the alias don't ovverride.

**Comment by Sandertv** [comment]
Hmmm I can't really reproduce this, the override works just fine for me... Could you try doing /gamemode c and tell me if it puts you in creative mode?

**Comment by SuperAlecs** [comment]
yes but the gamemode command is of pocketmine is not on essentials plugin

**Comment by Sandertv** [comment]
Aha I think I get what you mean now!

**Comment by Sandertv** [comment]
Actually, the gamemode command is in the essentials plugin, that's what the command overrides.

**Comment by Sandertv** [comment]
To be honest I'm not entirely sure how to fix your problem...

**Comment by SuperAlecs** [comment]
ok don't worry thanks for your help

**Comment by Sandertv** [comment]
I understand what you mean now completely, and I think your problem started since 0.16 no?

**Comment by SuperAlecs** [comment]
No it started on 0.15 no in 0.16

**Comment by Muqsit** [comment]
@SuperAlecs as of pmmp/gen 0.16, you will need to add `$player->sendCommandData()`
under PlayerJoinEvent in your permission-based plugin (eg: PurePerms).

**Comment by Sandertv** [comment]
That's what I was thinking, but he said it started in 0.15?

**Comment by Muqsit** [comment]
well, I have had no problems in 0.15 :/

**Comment by Sandertv** [comment]
Same here though, I'm not exactly sure of it.

**Comment by Sandertv** [comment]
I can't reproduce this, it all works the way it's supposed to...

**Comment by Muqsit** [comment]
same

**Comment by SOF3** [comment]
PMMP version and other plugins you are using?

@Sandertv what's the difference between Probably Bug and Unconfirmed...

Using `/EssentialsPE:gm` works?

**Comment by Sandertv** [comment]
They were both not made by me @SOF3, that's why I added both the tags.

**Comment by Sandertv** [comment]
I did notice that /r did not seem to work when players used /tell, (perhaps /msg as well?) so I assume that is part of your issue.

**Comment by kenygamer** [comment]
@Sandertv Oh, regarding that, please add an option to enable or disable *Override* in the *config*.
> ☆∂єvєlσρєя☆
кєνιи αи∂яєωѕ
